DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -45,10 +45,12 @@ struct NetworkProtectionLocationStatusModel {
         case .location(let location):
             let countryLabelsModel = NetworkProtectionVPNCountryLabelsModel(country: location.country, useFullCountryName: true)
             if let city = location.city {
-                title = UserText.netPVPNSettingsLocationSubtitleFormattedCityAndCountry(
+                let formattedCityAndCountry = UserText.netPVPNSettingsLocationSubtitleFormattedCityAndCountry(
                     city: city,
                     country: countryLabelsModel.title
                 )
+
+                title = "\(countryLabelsModel.emoji) \(formattedCityAndCountry)"
```

Comment:
This should ideally go into the localized string. We're doing this elsewhere so I've reused that pattern here, but in this form it won't handle RTL languages very well. I'll look at changing this before it gets merged.

Reply:
I’m assuming from Asana that you want this merged to make the release, so perhaps tackle this in a follow-up.